### DISCUSSION TITLE
Label for NumericUpDown?

### DISCUSSION THREAD

**Discussion by seikosantana** [AvaloniaCommunity/Material.Avalonia#299] [Q&A]
Hi, I tried using `assists:TextFieldAssist.Label` on a `NumericUpDown` but it doesnt seem to make any labels like it would on a TextBox. Is there a way to get the label just like in TextBox so it would look consistent?

**Comment by SKProCH** [comment]
Currently this is doesn't available yet. I've created an issue to track the progress: #301
